### PR TITLE
Shooting firework effect

### PR DESCRIPTION
**Pull request by MrMelon54**
Still in progress so please don't merge yet
You can still take a look though

### CODE REVIEW

**Comment by MrMelon54** [comment]
adds #1

File: fireworks/fireworks.go [hunk 1]
```diff
@@ -36,26 +37,45 @@ type model struct {
...
-			Physics: harmonica.NewProjectile(
-				harmonica.FPS(framesPerSecond),
-				harmonica.Point{X: x, Y: y},
-				harmonica.Vector{X: math.Cos(float64(i)) * v, Y: math.Sin(float64(i)) * v / 2},
-				harmonica.Vector(harmonica.TerminalGravity),
+			Physics: simulation.NewProjectile(
+				simulation.FPS(framesPerSecond),
+				simulation.Point{X: x, Y: y},
+				simulation.Vector{X: math.Cos(float64(i)) * v, Y: math.Sin(float64(i)) * v / 2},
+				simulation.Vector(simulation.TerminalGravity),
```
Comment (maaslalani) [inline fireworks/fireworks.go:73]: How come we are removing `harmonica` here?

I actually moved all the internal code to harmonica https://github.com/charmbracelet/harmonica/pull/6 for confetty to make use of it 😄

Comment (MrMelon54) [reply fireworks/fireworks.go:73]: Probably would be better to just expose the velocity and acceleration in harmonica then?

Comment (MrMelon54) [reply fireworks/fireworks.go:73]: but as I said this PR isn't complete yet so just wait while I make it work fully

Comment (MrMelon54) [reply fireworks/fireworks.go:73]: would be great if you could send PR to harmonica to add functions to get the velocity and acceleration of a projectile

Comment (maaslalani) [reply fireworks/fireworks.go:73]: https://github.com/charmbracelet/harmonica/pull/9

Comment (MrMelon54) [reply fireworks/fireworks.go:73]: awesome

Comment (maaslalani) [reply fireworks/fireworks.go:73]: Btw merge the harmonica PR

**Comment by maaslalani** [comment]
Changes merged in https://github.com/maaslalani/confetty/pull/9

**Pull request closed without merging**